src/scripts/main.js

```diff
@@ -354,7 +354,23 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const calculateAge = (born, died) => died - born;
```

Comment:
The `calculateAge` function correctly calculates the age by subtracting the birth year from the death year. Ensure that the data provided for `born` and `died` is always valid and that `died` is greater than or equal to `born` to avoid negative ages.